src/gwt/src/org/rstudio/studio/client/workbench/commands/Commands.cmd.xml

```diff
@@ -789,6 +790,7 @@ well as menu structures (for main menu and popup menus).
          <shortcut refid="setWorkingDir" value="Ctrl+Shift+H" />
          <shortcut refid="synctexSearch" value="Cmd+F8" />
          <shortcut refid="checkSpelling" value="F7" />
+         <shortcut refid="wordCount" value="Ctrl+Shift+D" />
```

Comment:
This conflicts with the Make Bookmark command on MacOS browsers. Since we have programmable keyboard shortcuts, a relatively low-use command like this one might be best left unbound (occasional users can use the menu and heavy users can bind to a shortcut of their choosing)